Util/Profiler/TokenFinder.php

```diff
@@ -23,7 +23,7 @@ class TokenFinder
      * @param Profiler $profiler
      * @param int      $defaultLimit
      */
-    public function __construct(Profiler $profiler, $defaultLimit)
+    public function __construct(?Profiler $profiler, $defaultLimit)
```

Comment:
Why?

Reply:
in the `find` method it has this check:

```php
if ($this->profiler === null) {
            return [];
        }
```

therefore my logic was that thing could be null too

Reply:
Yes but this check is added by you.
Before this parameter was mandatory.

Why does it need to be optional now?